Source/DafnyStandardLibraries/examples/Helpers.dfy

```diff
@@ -0,0 +1,19 @@
+/*******************************************************************************
+ *  Copyright by the contributors to the Dafny Project
+ *  SPDX-License-Identifier: MIT
+ *******************************************************************************/
+
+module Helpers {
+  import opened DafnyStdLibs.Wrappers
+    // TODO: consider tweaking /testContracts to support this use case better.
```

Comment:
+1, and would prefer an issue to a TODO, but don't want to block on it

Reply:
I just copied this comment over from where `Helpers` was originally defined, but here's a new issue for it: https://github.com/dafny-lang/dafny/issues/4786